caffe2/operators/max_pool_with_index.h

```diff
@@ -0,0 +1,48 @@
+#include <cfloat>
```

Comment:
So... I assume there is no CPU implementation for this op? It is probably fine since we don't really need to do maxpool with index if we do training (which is most likely going to be GPUs), but I wonder if this is something we should have for completeness.